### DISCUSSION TITLE
Allow users without an account to save posts and comments

### DISCUSSION THREAD

**Discussion by Outlet2048** [Docile-Alligator/Infinity-For-Reddit#900] [Ideas]
It is possible to follow subreddits without an account (which is amazing!) and I'd like to see the same functionality for saving comments and posts :)

**Comment by RickyM7** [comment]
Yes, its a really nice and userful feature. Maybe the thing I miss the most when using the app. (Sorry for my bad english)

**Reply by ghost** [reply]
Using two clients for the same.
Stealth for saving post and comments.
Infinity for everything else.
